.gitignore

```diff
@@ -0,0 +1,4 @@
+# .gitignore
```

Comment:
- [ ] Files that are not part of the project please add to the `.gitignore` files such as folder `hint-report`